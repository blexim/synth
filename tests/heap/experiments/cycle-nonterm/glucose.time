WARNING! DIMACS header mismatch: wrong number of variables.
Command exited with non-zero status 10
1.58user 0.05system 0:01.65elapsed 99%CPU (0avgtext+0avgdata 118028maxresident)k
0inputs+16outputs (0major+31899minor)pagefaults 0swaps
