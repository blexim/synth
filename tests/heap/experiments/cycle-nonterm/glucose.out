c
c This is glucose 4.0 --  based on MiniSAT (Many thanks to MiniSAT team)
c
c ========================================[ Problem Statistics ]===========================================
c |                                                                                                       |
c |  Number of variables:        158472                                                                   |
c |  Number of clauses:          633079                                                                   |
c |  Parse time:                   0.19 s                                                                 |
c |                                                                                                       |
c | Preprocesing is fully done
c |  Eliminated clauses:           6.16 Mb                                                                |
c |  Simplification time:          1.27 s                                                                 |
c |                                                                                                       |
c ========================================[ MAGIC CONSTANTS ]==============================================
c | Constants are supposed to work well together :-)                                                      |
c | however, if you find better choices, please let us known...                                           |
c |-------------------------------------------------------------------------------------------------------|
c |                                |                                |                                     |
c | - Restarts:                    | - Reduce Clause DB:            | - Minimize Asserting:               |
c |   * LBD Queue    :     50      |   * First     :   2000         |    * size <  30                     |
c |   * Trail  Queue :   5000      |   * Inc       :    300         |    * lbd  <   6                     |
c |   * K            :   0.80      |   * Special   :   1000         |                                     |
c |   * R            :   1.40      |   * Protected :  (lbd)< 30     |                                     |
c |                                |                                |                                     |
c ==================================[ Search Statistics (every  10000 conflicts) ]=========================
c |                                                                                                       |
c |          RESTARTS           |          ORIGINAL         |              LEARNT              | Progress |
c |       NB   Blocked  Avg Cfc |    Vars  Clauses Literals |   Red   Learnts    LBD2  Removed |          |
c =========================================================================================================
c last restart ## conflicts  :  123 130 
c =========================================================================================================
c restarts              : 9 (168 conflicts in avg)
c blocked restarts      : 0 (multiple: 0) 
c last block at restart : 0
c nb ReduceDB           : 0
c nb removed Clauses    : 0
c nb learnts DL2        : 119
c nb learnts size 2     : 75
c nb learnts size 1     : 12
c conflicts             : 1514           (954 /sec)
c decisions             : 12564          (0.00 % random) (7914 /sec)
c propagations          : 775059         (488226 /sec)
c conflict literals     : 31493          (20.93 % deleted)
c nb reduced Clauses    : 28
c CPU time              : 1.5875 s

s SATISFIABLE
